eos/rare-b-decays/b-to-kstar-ll-gvdv2020_TEST.cc

```diff
@@ -95,40 +95,175 @@ class BToKstarDileptonGvDV2020Test :
             p["b->smumu::Re{c10'}"] = 4;
             p["b->smumu::Im{c10'}"] = 3.5;
 
-            Options oo;
-            oo.declare("model",                   "WET");
-            oo.declare("tag",                     "GvDV2020");
-            oo.declare("nonlocal-formfactors",    "GvDV2020");
-            oo.declare("form-factors",            "BSZ2015");
-            oo.declare("l",                       "mu");
-            oo.declare("q",                       "d");
+            Options oo
+            {
+                {"model",                   "WET"},
+                {"tag",                     "GvDV2020"},
+                {"nonlocal-formfactors",    "GvDV2020"},
+                {"form-factors",            "BSZ2015"},
+                {"l",                       "mu"},
+                {"q",                       "d"}
+            };
 
             static const double eps = 1e-5;
             static const double q2 = 6.0;
 
             BToKstarDilepton d(p, oo);
             auto amps = d.amplitudes(q2);
 
-            TEST_CHECK_RELATIVE_ERROR(real(amps.a_long_left),  -1.20243e-10, eps);
-            TEST_CHECK_RELATIVE_ERROR(imag(amps.a_long_left),  -6.20413e-11, eps);
+            // TEST_CHECK_RELATIVE_ERROR(real(amps.a_long_left),  -1.20243e-10, eps);
```

Comment:
Why are these commented out in a non-wip commit?

Reply:
They shouldn't!